src/p10/ekb/p10_scom_addr.C

```diff
@@ -815,13 +815,18 @@ extern "C"
              ( getEndpoint() == PSCOM_ENDPOINT )  &&   // 0x1
              ( getRingId() >= OMI0_RING_ID )      &&   // 0x5
              ( getRingId() <= OMI1_RING_ID )      &&   // 0x6
-             ( getSatId()  == MC_SAT_ID0 )        &&   // 0x0 (DL)
-             ( getSatOffset() >= 0 )              &&   // shared regs 0-15
-             ( getSatOffset() <= 15 ) )
+             ( getSatId()  == MC_SAT_ID0 ) )
         {
-            l_omicTarget = true;
+             if (((getSatOffset() >= 16) &&             // 16:31 (subchannel 0)
```

Comment:
Looking at the EKB version of this file, lines 800 thru' 806 is no longer present. Its commented out. So could you test with that change instead? So to me it looks like a case of phal using an older outdated EKB file.